core/model/src/main/java/org/eclipse/sw360/antenna/model/artifact/facts/java/MavenCoordinates.java

```diff
@@ -69,7 +69,7 @@ public String getType() {
 
     @Override
     protected PackageURLBuilder addPurlFacts(PackageURLBuilder builder) {
-        return builder.withNamespace(getGroupId());
+        return builder.withNamespace(getGroupId()).withName(getArtifactId());
```

Comment:
what about the `version`? It is handled somewhere else?

Reply:
See above